lms/djangoapps/support/static/support/jsx/logged_in_user.jsx

```diff
@@ -10,7 +10,11 @@ function LoggedInUser({ userInformation, setErrorState, zendeskApiHost, submitFo
   if (userInformation.enrollments) {
     courseElement = (<div>
       <label className="label-course" htmlFor="course">{gettext('Course Name')}</label>
-      <select className="form-control select-course" id="course">
+      <select className="form-control select-course" id="course" value={userInformation.course_id}>
+        <option key="select-course" value="">--------</option>
```

Comment:
I think we don't need this option, by default we can select "not specific to a course" option. That will also reduce the complexity or errors checking.

Reply:
remove the [`else`](https://github.com/edx/edx-platform/blame/b8ed7bb21dccd2047aa2e9f8a43e06c2f68befec/lms/djangoapps/support/static/support/jsx/logged_in_user.jsx#L26-L29)  part, as we are making this required filed.

Reply:
@tasawernawaz I don't think we need to remove anything her, we can just remove the `Optional` text if learner is not enrolled in any course.